packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -210,45 +212,17 @@ describe("Garbage Collection Tests", () => {
         });
     });
```

Comment:
We should get a test added that confirms the fix for the bug about wrongly updating the metadata. If you can't do it today no worries.

Reply:
Can you please create a task to do that as a follow up? Don't think I can get to it.